cypress/integration/task/aborted-task.ts

```diff
@@ -0,0 +1,28 @@
+describe("Task page for an aborted task", () => {
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    cy.preserveCookies();
+  });
+
+  it("Link to failing task in task metadata should exist for aborted tasks", () => {
```

Comment:
Test names should read like a story so its more natural to read

`it("Should link to ...`